### PR TITLE
Documentation modal/rating/taost/color

### PR DESCRIPTION
**Pull request by sanu29**
image
image
image
image

### CODE REVIEW

File: Utils/variables.css [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
      --primary-color       : #04566E ;
      --secondary-one-color : #339EA0;
-     --secondary-two-color : #FCF388;
-     --secondary-nav-color : #E1E2E2;
+     --secondary-two-color : rgb(252, 243, 136);
```
Comment (dev-prabhat) [inline Utils/variables.css]: try to be consistence with color code

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Looks good just change that color code

**Review by dev-prabhat — APPROVED** [review]
Good to merge

**Review by aiman-ansari — APPROVED** [review]
Everything looks good to merge.

**Pull request merged**